### PR TITLE
Mod porting to 1.19.2

### PR DESCRIPTION
**Pull request by Daniel-Mendes**
**Informations**
Minecraft version: 1.19.2
Forge version: 43.1.47

**Features not implemented**
- Moving caterpillar blocks one by one
- Particles when drilling

**New features implemented**
- Tutorial button for each caterpillar GUI
- Storage block fully implemented
- Reinforcement GUI use new buttons to activate replacement
- New logo for mod

### CODE REVIEW

**Review by The-Fireplace — CHANGES_REQUESTED** [review]
Please adjust the logo file to have one T instead of two. and if that logs directory isn't ignored already, add it.

**Comment by The-Fireplace** [comment]
Since a good chunk of this is rewritten, let's go ahead and change to a more modding-friendly license. Probably MIT or MPL-2.0.

File: src/main/resources/META-INF/mods.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+modLoader="javafml"
+loaderVersion="${loaderVersionRange}"
+license="GNU LGPLv3"
+issueTrackerURL="https://github.com/The-Fireplace/Simply-Caterpillar/issues"
```
Comment (The-Fireplace) [inline src/main/resources/META-INF/mods.toml:4]: Let's change the github links. The old ones technically work but we probably shouldn't rely on them forever.
```suggestion
issueTrackerURL="https://github.com/The-Fireplace-Minecraft-Mods/Simply-Caterpillar/issues"
```

Comment (Daniel-Mendes) [reply src/main/resources/META-INF/mods.toml]: Change made in [commit 3ce1a1f](https://github.com/The-Fireplace-Minecraft-Mods/Simply-Caterpillar/pull/4/commits/3ce1a1fe328b2b26d9d65c87b37920deedae9bca)

File: src/main/resources/assets/simplycaterpillar/lang/en_US.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
```
Comment (The-Fireplace) [inline src/main/resources/assets/simplycaterpillar/lang/en_US.json:1]: This file name should be all lowercase.

File: src/main/resources/META-INF/mods.toml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+modLoader="javafml"
+loaderVersion="${loaderVersionRange}"
+license="GNU LGPLv3"
```
Comment (The-Fireplace) [inline src/main/resources/META-INF/mods.toml:3]: This can be updated to say MIT

**Review by The-Fireplace — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/simplycaterpillar/lang/en_US.json (hunk 2 above)
Comment (Daniel-Mendes) [reply src/main/resources/assets/simplycaterpillar/lang/en_US.json]: Fixed in commit https://github.com/The-Fireplace-Minecraft-Mods/Simply-Caterpillar/pull/4/commits/6a3dfb1b52f7d003d0d2b9d4846783319089f11d

File: src/main/resources/META-INF/mods.toml (hunk 3 above)
Comment (Daniel-Mendes) [reply src/main/resources/META-INF/mods.toml]: Changed in commit https://github.com/The-Fireplace-Minecraft-Mods/Simply-Caterpillar/pull/4/commits/77f6b7c222da62089d04745baa56b7c1047e442e

**Pull request merged**